src/com/goide/psi/impl/GoPsiImplUtil.java

```diff
@@ -1130,6 +1130,15 @@ public static boolean isCImport(@NotNull GoImportSpec importSpec) {
     return GoConstants.C_PATH.equals(importSpec.getPath());
   }
 
+  public static boolean isRelativeImport(@NotNull GoImportSpec importSpec) {
+    return GoImportReferenceSet.isRelativeImport(importSpec.getPath());
```

Comment:
Well, creating method for PsiElement which just invoke some other reference-specific method is strange.